src/components/Board/PanZoomRotate.js

```diff
@@ -50,13 +50,25 @@ const PanZoomRotate = ({ children }) => {
   }, [config, setDim]);
 
   const onWheel = (e) => {
-    if (e.altKey || e.ctrlKey) {
+    if (e.altKey) {
       return;
     }
 
     const scaleMult = (e.deltaY < 0 ? -3 : 3 * dim.scale) / 20;
 
     setDim((prevDim) => {
+      // On a trackpad, the pinch and pan events are differentiated by the crtlKey value.
+      // On a pinch gesture, the ctrlKey is set to true, so we want to have a scaling effect.
+      // If we are only moving the fingers in the same direction, a pan is needed.
+      // Ref: https://medium.com/@auchenberg/detecting-multi-touch-trackpad-gestures-in-javascript-a2505babb10e
+      if (!e.ctrlKey) {
```

Comment:
It will break the navigation with the mouse though, as they don't have this `ctrlKey` set to `true` when zooming I guess @jrmi ...

Reply:
I don't really like that, but I don't see any other option: https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/platform can help you to detect the platform and enable this feature only for mac. It means other mac users without multi-touch trackpad will not be able to zoom until they press the ctrl key.

Reply:
May be better using https://github.com/bestiejs/platform.js ?

Reply:
Or not. As you feel.